File: simcoe/src/main/java/com/prolificinteractive/simcoe/ConsoleOutput.java
package com.prolificinteractive.simcoe;

import android.util.Log;

/**
 * The console output.
 */
public class ConsoleOutput implements Output {

  private final String SIMCOE_LOG_TAG = "Simcoe";

  @Override public void print(final String message) {
    Log.v(SIMCOE_LOG_TAG, message);
  }
}

File: simcoe/src/main/java/com/prolificinteractive/simcoe/ConsoleOutput.kt
package com.prolificinteractive.simcoe

import android.util.Log

/**
 * The console output.
 */
internal class ConsoleOutput : Output {
  private val SIMCOE_LOG_TAG = "Simcoe"

  override fun print(message: String) {
    Log.v(SIMCOE_LOG_TAG, message)
  }
}
